README.md

```diff
@@ -247,14 +263,14 @@ The following files have the code that would be of interest to you.
 
     * Interactive:
     ```java
-    mMultipleAccountApp.acquireToken(getActivity(), getScopes(), getAuthInteractiveCallback());
-    ```
-
+    mMultipleAccountApp.acquireToken(getActivity(), getScopes(),
+                                     getAuthInteractiveCallback());
+```

```

Comment:
The metacharacters are rendering in the previewer

![Capture](https://user-images.githubusercontent.com/990063/65784323-a4a76d00-e106-11e9-8a78-f94a1ba977a4.PNG)

Reply:
I suggested some edits here: https://github.com/Azure-Samples/ms-identity-android-java/pull/11